class.json-api-endpoints.php

```diff
@@ -267,11 +267,28 @@ function input( $return_default_values = true, $cast_and_filter = true ) {
 			break;
 		}
 
-		if ( !$cast_and_filter ) {
-			return $return;
+		if ( isset( $this->api->query['force'] ) && 'secure' === $this->api->query['force'] ) {
+			if ( isset( $return['secure_key'] ) ) {
+				$post_body = $this->get_secure_body( $return['secure_key'] );
+				$this->api->post_body = $post_body;
```

Comment:
If nothing else is being done with $body_post, why not just set $this->api->post_body = $this->get_secure_body( $return['secure_key'] ); and eliminate line 273?